packages/smooth_app/lib/cards/data_cards/image_upload_card.dart

```diff
@@ -56,6 +62,9 @@ class _ImageUploadCardState extends State<ImageUploadCard> {
           imageUrl: _image.uri,
         );
 
+        print('barcode:' + widget.product.barcode);
```

Comment:
Please remove the `print` statements from code to be reviewed.

Reply:
ok